src/button/button.scss

```diff
@@ -7,6 +7,44 @@ $iui-button-padding: $iui-xs * 4;
 $iui-button-padding-small: $iui-xs * 2;
 $iui-button-padding-large: $iui-xs * 6;
 
+@mixin iui-button-icon {
+  > .iui-icon:not(.iui-user-icon) {
+    width: $iui-icons-default;
+    height: $iui-icons-default;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: fill $iui-speed-fast ease-out;
+    }
+  }
+
+  > .iui-icon {
+    @include themed {
+      fill: t(iui-icons-color-actionable);
+    }
+  }
+
+  &:enabled:hover,
```

Comment:
Is it possible to remove `:enabled` from all of the button's CSS?

Reply:
If the specificity of the hover selector stays low, then disabled styling will take preference if it comes later:
```scss
&:hover { ... }
&:disabled { ... }
```

Reply:
Removed all `:enabled` from buttons.